hack/common.sh

```diff
@@ -29,12 +35,20 @@ readonly OS_SCRATCH_IMAGE_COMPILE_TARGETS=(
 )
 readonly OS_IMAGE_COMPILE_BINARIES=("${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]##*/}" "${OS_IMAGE_COMPILE_TARGETS[@]##*/}")
 
-readonly OS_CROSS_COMPILE_PLATFORMS=(
+OS_CROSS_COMPILE_PLATFORMS=(
   linux/amd64
   darwin/amd64
   windows/amd64
   linux/386
 )
+if [[ "$(os::build::host_platform)" == "linux/ppc64le" ]]; then
```

Comment:
Unless the issues actually make it impossible to do this cross-compilation on platforms other than `ppc64le`, we should just keep the list of targets full and static -- having this added on certain hosts kind of breaks the whole point of `hack/build-cross.sh`.